Gemfile

```diff
@@ -28,5 +29,4 @@ gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.0" if Gem.win_platform?
 
-
-gem 'cssminify'
\ No newline at end of file
+gem 'cssminify'
```

Comment:
Does this work on github pages?

Reply:
Honestly not sure! It was in the Gemfile before, though.